blockdevice/blockdevice.go

```diff
@@ -159,6 +181,81 @@ type FileSystemInformation struct {
 	MountPoint []string
 }
 
+// CapacityInformation holds the capacity related information for the device
+type CapacityInformation struct {
+	// Storage is the storage capacity of this blockdevice
+	// in bytes
+	Storage uint64
+
+	// PhysicalBlockSize is the physical block size in bytes
```

Comment:
@pawanpraka1 , should the `PhysicalBlockSize`, `LogicalBlockSize` and `HardwareSectorSize` be a part of `CapacityInformation` or `DeviceInformation`

Reply:
it should be part of DeviceInformation as they are the attributes of the device.

Reply:
Should `PhysicalSectorSize` and `LogicalSectorSize` part of this?

Reply:
They are same as `PhysicalBlockSize` and `LogicalBlockSize` given by the system. I have added the comments describing how it is used.